Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -130,5 +156,59 @@ public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_
     return self
   }
 % end
+  
+  public func lineComment(_ text: String) -> Self {
+    
+    let textWithSlashesPieces = prefixLines(of: text, with: "//")
+      .map { TriviaPiece.lineComment("\($0)") }
+    return addCommentPiece(commentPieces: textWithSlashesPieces)
+  }
+  
+  /// A documentation block comment, starting with '/**' and ending with '*/'.
+  /// - Parameter text:  inserted between /*  and  */.
+  /// - Returns: a `${type.buildable()}` with the added line comment.
+  public func blockComment(_ text: String) -> Self {
+    let piece = TriviaPiece.blockComment("/* \(text) */")
+    return addCommentPiece(commentPieces: [piece])
+  }
+  
+  /// A documentation line comment, starting with '/**' and ending with '*/'.
+  /// - Parameter text: comment to be inserted after ///
+  /// - Returns: a `${type.buildable()}` with the added documentation line comment.
+  public func docLineComment(_ text: String) -> Self {
+    let textWithSlashesPieces = prefixLines(of: text, with: "//")
+      .map { TriviaPiece.docLineComment("\($0)") }
+    return addCommentPiece(commentPieces: textWithSlashesPieces)
+  }
+
+  
+  /// A documentation block comment, starting with '/**' and ending with '*/'.
+  /// - Parameter text: comment to be inserted between /** and '*/.
+  /// - Returns: a `${type.buildable()}` with the added  documentation block comment.
+  public func docBlockComment(_ text: String) -> Self {
+    let piece = TriviaPiece.docBlockComment("/** \(text) */")
+    return addCommentPiece(commentPieces: [piece])
+  }
+  
+  private func addCommentPiece(commentPieces: [TriviaPiece]) -> Self {
+    var newSelf = self
+    newSelf.itemLeadingTriviaPieces.append(contentsOf: commentPieces)
+    return newSelf
+  }
+  
+  /// Extracts Lines from a given text, and add a prefix to it
+  /// - Parameters:
+  ///   - text: Text which its lines will be prefixed
+  ///   - prefix: prefix for each line
+  /// - Returns: an array of new string, which has prefixed lines
+  private func prefixLines(of text: String, with prefix: String) -> [String] {
+    
+    let prefixedLines = text
+      .split(whereSeparator: \.isNewline)
+      .map {"\(prefix) \($0)"}
+    
+    return prefixedLines
+  }
```

Comment:
A little bit of a nitpick but I think this is a general helper method and not something that needs to access instance members of `*Buildable`. Thus, I think it would make sense to make it a free function instead of an instance function.